lib/src/generator/template_data.dart

```diff
@@ -103,7 +103,11 @@ abstract class TemplateData<T extends Documentable> extends TemplateDataBase {
 
   String? get belowSidebarPath => self.belowSidebarPath;
 
-  String _layoutTitle(String name, String kind, {required bool isDeprecated}) =>
+  String _layoutTitle(
+    String name,
+    String? kind, {
```

Comment:
Could make this a named parameter, then you don't need to send `null` over in the getter.

Reply:
Good call. Done.